services.py

```diff
@@ -4,61 +4,75 @@
 
 EON = os.path.isfile('/EON')
 
+RESERVED_PORTS = [8022, 22]  # sshd
+STARTING_PORT = 8001
+
+
+cur_port = STARTING_PORT
+def new_port():
```

Comment:
Don't like the global. Why not keep the port argument to `Service`, but build the dict dynamically.

E.g. something like
```python
services = {
  "roadCameraState": (True, 20., 1),
  "sensorEvents": (True, 100., 100),
}
service_list = {name: Service(i + 8000, *vals) for i, (name, vals) in enumerate(sevices.items())}
```

Reply:
I couldn't figure out how to create a function that could skip the reserved ports while providing continuous ports just with the index without knowing the previous port (or back calculating it starting from 8001, which takes exponentially more time as we add services) so I relied on the handy walrus operator to do this.

Reply:
I guess since you don't care what the ports are we could just add an offset to the ports that happen to be reserved, something like the length of the `services` dict so that it doesn't collide with the last service port.